packages/svelte/src/internal/shared/attributes.js

```diff
@@ -40,3 +40,46 @@ export function clsx(value) {
 		return value ?? '';
 	}
 }
+
+/**
+ * @param {any} clazz
+ * @param {string|null} [hash]
+ * @param {Record<string,boolean>} [classes]
+ * @returns {string|null}
+ */
+export function to_class(clazz, hash, classes) {
+	let class_name = clazz == null ? '' : '' + clazz;
```

Comment:
please, anything but `clazz` 😆 

```suggestion
export function to_class(value, hash, classes) {
	let class_name = value == null ? '' : '' + value;
```